packages/colorpicker/lib/colorpicker.dart

```diff
@@ -0,0 +1,116 @@
+library colorpicker;
+
+import 'package:colorpicker/constants/colors.dart';
+import 'package:colorpicker/widgets/color_compare.dart';
+import 'package:colorpicker/widgets/slider.dart';
+import 'package:flutter/material.dart';
+
+class ColorPicker extends StatefulWidget {
+  const ColorPicker({
+    super.key,
+    required this.currentColor,
+    required this.onColorChanged,
+  });
+
+  final Color currentColor;
+  final void Function(Color) onColorChanged;
+
+  @override
+  State<ColorPicker> createState() => _ColorPickerState();
+}
+
+class _ColorPickerState extends State<ColorPicker> {
+  final colors = DisplayColors.colors;
+  Color newColor = Colors.black;
+  double opacity = 1.0;
+
+  void callback(Color color, double op) {
+    setState(() {
+      opacity = op;
+    });
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    newColor = widget.currentColor;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.all(26),
+      alignment: Alignment.center,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(16),
+      ),
+      child: Column(
+        children: [
+          ColorCompare(
+            currentColor: widget.currentColor,
+            newColor: newColor.withOpacity(opacity),
+          ),
+          const SizedBox(
+            height: 10,
+          ),
+          GridView.count(
+            childAspectRatio: 1.4,
+            crossAxisCount: 4,
```

Comment:
4.0

Reply:
I think `crossAxisCount` accepts only integers.. so leaving it as it is